src/devices/Display/samples/Large4Digit7SegmentDisplay.sample.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Display.Enums;
+using Iot.Device.Display.LedDriver;
+using Iot.Device.Display.SevenSegment;
+using Iot.Device.Display.SevenSegment.Enums;
+
+namespace SevenSegmentSample
+{
+    class Program
+    {
+        const string SupportedCharacters = "0123456789aAbBcCdDeEfFgGhHiIjJlLnNoOpPrRsStuUyYzZ-=_|°[]     ";
+
+        static void Main(string[] args)
+        {
+            // Initialize display (busId = 1 for Raspberry Pi 2 & 3)
+            var display = new Large4Digit7SegmentDisplay(I2cDevice.Create(new I2cConnectionSettings(busId: 1, Ht16k33.DefaultI2cAddress)))
+            {
+                // Set max brightness
+                Brightness = Ht16k33.MaxBrightness
+            };
+
+            // Write "Iot" on the display
+            display.Write("Iot");
+            // Send buffer to the device
+            display.Flush();
+
+            // Wait 2 seconds
+            Thread.Sleep(2000);
+
+            // Change 'o' to '°' in "IoT"
+            display[1] = (Segment)FontHelper.GetCharacter('°');
```

Comment:
can `°` be used directly in Write? Assuming this is just showcasing the API please at least mention that in the comment. Reading the comment it slightly implies special characters won't work and I think they do

Reply:
As a matter of fact it can, it is listed in the `SupportedCharacters` constant in line 18 which is iterated and written to the display between lines 159 and 166.

I added this line because it showcased the address operator and the `FontHelper.GetCharacter` method, and maybe also that the `Font` and `Segment` enums are interchangeable.